core/toolbox/toolbox.ts

```diff
@@ -299,9 +298,8 @@ export class Toolbox extends DeleteArea implements IAutoHideable,
     if (!handled && this.selectedItem_) {
       // TODO(#6097): Figure out who implements onKeyDown and which interface it
       // should be part of.
-      const untypedItem = this.selectedItem_ as AnyDuringMigration;
-      if (untypedItem.onKeyDown) {
-        handled = untypedItem.onKeyDown(e);
+      if ((this.selectedItem_ as any).onKeyDown) {
+        handled = (this.selectedItem_ as any).onKeyDown(e);
```

Comment:
It doesn't appear that `ISelectableToolboxItem` has an `.onKeyDown`, so I'm slightly unclear about how this is expected to work.  Does the interface need to be modified?  (Noting that it would be a breaking change.)